package-overrides/npm/parse@1.6.5.json

```diff
@@ -0,0 +1,8 @@
+{
+  "dependencies": {
+    "babel-runtime": "^5.8.20"
+  },
+  "devDependencies": {},
+  "main": "dist/parse-latest.js",
+  "files": ["dist/parse-latest.js"]
```

Comment:
It's also advisable to leave this property out in case users want to use other files?